client/lib/cart-values/index.js

```diff
@@ -65,6 +65,10 @@ function isFree( cart ) {
 	return cart.total_cost === 0 && ! cartItems.hasFreeTrial( cart );
 }
 
+function isFreeTrial( cart ) {
```

Comment:
There actually was a hasFreeTrial already in the lib, so I switched to using that and removed the new isFreeTrial altogether in: 731034c4c16c4d10ab7cefed85fd0e182766bce5